packages/synapse-react-client/src/components/widgets/query-filter/FacetFilterControls.tsx

```diff
@@ -107,10 +114,33 @@ export const applyChangesToRangeColumn = (
   facet.columnMin = values[0]
   facet.columnMax = values[1]
   const changedFacet = convertFacetColumnRangeRequest(facet)
-  const result = patchRequestFacets(changedFacet, lastRequest)
+  const result = patchRequestFacets(
+    changedFacet,
+    lastRequest?.query?.selectedFacets,
+  )
   onChangeFn(result)
 }
 
+export const applyCombinedChangesToRangeColumn = (
```

Comment:
Update the ranges to both facet filters based on the values from the CombinedRangeFacetFilter component